scripts/index.js

```diff
@@ -105,36 +109,19 @@ fs
     spinner.succeed();
 
     return new Promise((resolve, reject) => {
-      const webpackCompiler = webpack(config);
-      webpackCompiler.apply(
-        new webpack.ProgressPlugin(() => {
-          if (!inProgress) {
-            spinner.start('Start webpack watch');
-            inProgress = true;
-          }
-        })
-      );
-
-      webpackCompiler.watch({}, (err, stats) => {
+      const options = {
```

Comment:
[Here](https://github.com/facebook/create-react-app/blob/ff19e0a6d22ba070f5b4c3b511edb4e8d4995686/packages/react-scripts/config/webpackDevServer.config.js#L22) is `react-scripts` webpackDevServer config file and it would be nice to just add there the configuration we need pretty much like I did for their webpack config.